common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftTestExtension.java

```diff
@@ -38,16 +39,20 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.Optional;
+import java.util.ServiceLoader;
 import java.util.stream.Stream;
 
 import static org.fusesource.jansi.Ansi.ansi;
 import static org.junit.platform.commons.util.AnnotationUtils.findAnnotatedFields;
 
-// TODO at this point, this class is close to becoming unreadable, and could use some refactoring
+// TODO at this point, this class is close to becoming unreadable, and could use some refactoring. 
+// Raised https://github.com/quarkus-qe/quarkus-openshift-test-suite/issues/108 for the refactoring.
```

Comment:
As stated in the comment, I've raised the https://github.com/quarkus-qe/quarkus-openshift-test-suite/issues/108 issue to do this.